<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Trier</class>
 <widget class="QDialog" name="Trier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>trier</string>
  </property>
  <widget class="QLineEdit" name="txtAttribut">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtV">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRechercher">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Rechercher</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>461</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Rechercher par </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Entrer La valeur</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Rechercher par : nom,pronom,massar,module,note</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
